text/0000-bionic-mixins.md

```diff
@@ -0,0 +1,85 @@
+# Meta
+[meta]: #meta
+- Name: Defining Mixins for io.buildpacks.stacks.bionic
+- Start Date: 2019-12-05
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+The Platform API specification allows stack maintainers to define stack mixins for a given stack.
+The Cloud Native Buildpacks project informally defines stack build and run images with ID `io.buildpacks.stacks.bionic` as containing exactly the set of Ubuntu packages in the `ubuntu:bionic` image on Docker Hub.
+This RFC proposes that we formalize the existing definition of `io.buildpacks.stacks.bionic` and define mixins on that stack to be Ubuntu 18.04 LTS packages. 
+
+In general, a stack is a contract that should be more like "Ubuntu Bionic" and less like "Platform XYZ's Re-distribution of Ubuntu Binoic."
+Stack images can still be provided by platforms/vendors, but they should use the standard CNB stack contracts where applicable.
+
+# Proposal
+[proposal]: #proposal
+
+This RFC proposes that we define `io.buildpacks.stacks.bionic` such that:
```

Comment:
Does it expand the scope too much to set a framework for adding new stacks under the `io.buildpacks.stacks` namespace or is that just a separate proposal. For instance, I think setting the `set=` pattern is probably more valuable then just defining what all the sets are in this proposal. @sclevine

Reply:
We could define `set=` for `io.buildpacks.stacks.*`, but it might be safer to hold off until we have additional stacks to define it for. The `set=` convention may be applicable to most base images that use OS packages today, but container base images could change in unexpected ways in the future.